app/actions/CompanyActions.ts

```diff
@@ -35,10 +35,10 @@ export const fetchAll = ({ fetchMore }: { fetchMore: boolean }) => {
   });
 };
 
-export function fetchAllAdmin() {
+export function fetchAllAdmin(semesterId: EntityId) {
   return callAPI<AdminListCompany[]>({
     types: Company.FETCH,
-    endpoint: '/bdb/',
+    endpoint: `/bdb/?semester_id=${semesterId}`,
```

Comment:
Does `/bdb?semester=${semesterId}` (without the `/` after `bdb`) work? I feel like that makes more sense when using query params

Reply:
yes, but it's faster to filter by semesterId

Reply:
I agree with using semesterId, I just meant that the `/` seems unnecessary
```suggestion
    endpoint: `/bdb?semester_id=${semesterId}`,
```

Reply:
Forget it, i think it is needed, cuz django or something